featuretools/feature_discovery/feature_discovery.py

```diff
@@ -0,0 +1,413 @@
+from itertools import combinations, permutations, product
+from typing import Dict, List, Type
+
+import woodwork.type_sys.type_system as ww_type_system
+from woodwork.column_schema import ColumnSchema
+from woodwork.logical_types import LogicalType
+from woodwork.table_schema import TableSchema
+
+from featuretools.feature_discovery.type_defs import ANY, Feature
+from featuretools.primitives.base.primitive_base import PrimitiveBase
+
+
+def column_to_keys(column_schema: ColumnSchema) -> List[str]:
+    """
+    Generate hashing keys from Columns Schema. For example:
```

Comment:
I think this description would be a good name for the function

Reply:
done